Update documentation (#98)
elastic/apm-agent-android: docs/setup.asciidoc

    }
}
----

<1> Initialize the Elastic APM agent once.

[float]
[[compile-and-run]]
=== Compile and run

All that's left is to compile and run your application.
That's it!

> After initializing the agent as shown above, you'll automatically create traces for all OkHttp network requests (including those created by tools that make use of OkHttp, such as Retrofit) and for all https://developer.android.com/reference/android/app/Activity[Activity] and https://developer.android.com/reference/androidx/fragment/app/Fragment[Fragment] starting methods.
> You'll also get crash reports sent automatically for when an unexpected error occurs in your app, and these events, as well as the traces already mentioned, will contain a "Session ID" that links the related events together in order to quickly get a clear view of any issue and its root cause.

Reviewer: What if we move this content and the APM app/config/manual links into a new section called something like, "What's next?"? Here's what I'm thinking:

```suggestion
[float]
[[whats-next]]
=== What's next?

After initializing the agent as shown above, your application will automatically create traces for all OkHttp network requests (including those created by tools that make use of OkHttp, like Retrofit) and all https://developer.android.com/reference/android/app/Activity[Activity] and https://developer.android.com/reference/androidx/fragment/app/Fragment[Fragment] starting methods.
In addition, you'll get automatic crash reports when an unexpected error occurs in your app.

All of these events will contain a "Session ID" that links related events together—allowing you to make sense of and diagnose any issues that arise. Head to the **APM app in {kib}** to start exploring your data.

If you need to customize the Elastic APM agent to your project's needs, see <<configuration,configuration>>. If you need to create your own custom transactions, see <<manual-instrumentation, manual instrumentation>>.
```

Author: I think it's great because as it is now, it seems like the setup process was cut off or something's missing, so the "What's next" title should help show where the setup ends and what they can do if that's not enough. Thanks!

> Head over to the **APM app in {kib}** to see your Android application events, or see <<configuration,configuration>> for details on how to tailor the Elastic APM agent to your project's needs.
> Optionally, you can take a look at <<manual-instrumentation, manual instrumentation>> in case you need to create your own transactions as well.

Reviewer: If you accept my above comment, then this content would move.

```suggestion
```

> Head over to the **APM app in {kib}** to see your Android application events, or see <<configuration,configuration>> for details on how to tailor the Elastic APM agent to your project's needs.
> Optionally, you can take a look at <<manual-instrumentation, manual instrumentation>> in case you need to create your own transactions as well.

Reviewer: If you accept my above comment, then this content would move.

```suggestion
```

> Head over to the **APM app in {kib}** to see your Android application events, or see <<configuration,configuration>> for details on how to tailor the Elastic APM agent to your project's needs.
> Optionally, you can take a look at <<manual-instrumentation, manual instrumentation>> in case you need to create your own transactions as well.

Reviewer: If you accept my above comment, then this content will move.

```suggestion
```
